version.props

```diff
@@ -1,8 +1,8 @@
 <Project>
   <PropertyGroup>
     <KoreBuildChannel>dev</KoreBuildChannel>
-    <VersionPrefix>2.1.0</VersionPrefix>
-    <VersionSuffix>preview3</VersionSuffix>
+    <VersionPrefix>2.2.0</VersionPrefix>
+    <VersionSuffix>preview1</VersionSuffix>
```

Comment:
Let's go with "alpha1" for now. Gives us more flexibility in the future. That's what we're doing for the runtime too, right?

Reply:
Well, we went with preview1 there: https://github.com/aspnet/Mvc/blob/dev/version.props#L4, so it might make sense to be consistent.

Reply:
Oh. That's not what we discussed last week. Was there a thread on this that I missed?

Reply:
cc @ryanbrandenburg

Reply:
When I asked what version we were branching to I was told 2.2.0-preview1 by @DamianEdwards.

Reply:
Ok.